performance-timeline/po-disconnect.html

```diff
@@ -30,15 +30,16 @@
 
   async_test(function (t) {
     var observer = new PerformanceObserver(
-        t.step_func(function(entryList, obs) {
-          checkEntries(entryList.getEntries(),
-            [{ entryType: "mark", name: "mark1"}]);
-          t.done();
+        t.step_func(function (entryList, obs) {
+          assert_unreached("This callback must not be invoked");
```

Comment:
Wouldn't this change break for chrome today?
I know we discussed updating the chrome implementation if there aren't too many callers depending on this, and updating the spec with takeRecords, so this change seems pre-mature (and would prevent importing these tests into chromium's auto-sync process for WPT)
https://github.com/w3c/performance-timeline/issues/66#issuecomment-277830576
@igrigorik WDYT?